### PR TITLE
Update npm-scripts.md

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: doc/misc/npm-scripts.md [hunk 1]
```diff
@@ -119,6 +119,8 @@ executing the scripts.  So, if your package.json has this:
 then you could run `npm start` to execute the `bar` script, which is
 exported into the `node_modules/.bin` directory on `npm install`.
 
+This is especially important for proper Windows support.
```
Comment (othiym23) [inline doc/misc/npm-scripts.md]: Needs more context. What aspect of this is important for Windows support? Also, "proper" is a content-free word in this context.

Comment (XhmikosR) [inline doc/misc/npm-scripts.md]: The thing is that people do `./node_modules/.bin/foo bar` and fails in scripts whereas `foo bar` works fine everywhere.

Comment (othiym23) [inline doc/misc/npm-scripts.md]: Then write that in the documentation. The current wording is vague and general enough to not communicate why it's important at all.

**Comment by XhmikosR** [comment]
Sorry, I deleted my fork since I have no time to reword this.

**Comment by othiym23** [comment]
Closing as abandoned.

**Pull request closed without merging**